app/assets/stylesheets/base/_variables.scss

```diff
@@ -21,13 +21,14 @@ $dark-gray: #333;
 $medium-gray: #999;
 $light-gray: #ddd;
 $hotline-bling-pink: #FA9ABA;
+$dark-pink: darken($hotline-bling-pink, 40%);
```

Comment:
Gray on pink doesn't look gray-t.

Black works fine. I can change them to that.